src/Jackett.Common/Definitions/newheaven.yml

```
@@ -65,6 +65,10 @@ caps:
     book-search: [q]
 
 settings:
+  - name: info_general
+    type: info
+    label: Layout Manager Settings
+    default: "Go to Usercenter->Einstellungen and scroll down to \"Layout Manager\" and ensure \"Comments\"-, \"Files\"- and \"Snatcher\"-Anzeige are set to \"anzeigen\" to ensure that Prowlarr can correctly detect the number of files, grabs, seeders and leechers."
```

```suggestion
    default: "Go to Usercenter->Einstellungen, scroll down to <b>Layout Manager</b> and ensure <b>Comments</b>-, <b>Files</b>- and <b>Snatcher</b>-Anzeige are set to <b>anzeigen</b>, so that the number of files, grabs, seeders and leechers can be correctly detected."
```

Slight tidy, and rephrasing so that it's not Jackett/Prowlarr-specific.

---

Ah, totally makes sense, sorry for that, thank you.